packages/xai/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@ai-sdk/xai",
+  "version": "1.0.0-canary.0",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "internal/dist/**/*",
+    "CHANGELOG.md"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "build:watch": "tsup --watch",
+    "clean": "rm -rf dist && rm -rf internal/dist",
+    "lint": "eslint \"./**/*.ts*\"",
+    "type-check": "tsc --noEmit",
+    "prettier-check": "prettier --check \"./**/*.ts*\"",
+    "test": "pnpm test:node && pnpm test:edge",
+    "test:edge": "vitest --config vitest.edge.config.js --run",
+    "test:node": "vitest --config vitest.node.config.js --run"
+  },
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./internal": {
+      "types": "./internal/dist/index.d.ts",
+      "import": "./internal/dist/index.mjs",
+      "module": "./internal/dist/index.mjs",
+      "require": "./internal/dist/index.js"
+    }
+  },
+  "dependencies": {
+    "@ai-sdk/provider": "1.0.0-canary.0",
+    "@ai-sdk/provider-utils": "2.0.0-canary.3"
```

Comment:
This needs to be exactly the current version in the repo or the changesets updates won't work.

Reply:
I just double checked this and above and both seem to match, am I missing something?

Reply:
ah yes, you're right, got confused by the low canary numbers